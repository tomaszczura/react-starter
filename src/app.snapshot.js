import React from 'react';
import renderer from 'react-test-renderer';
import { fromJS } from 'immutable';
import App from './app';

const { expect } = global;

describe('App snapshot', () => {
  test('initial render', () => {
    const component = renderer.create(<App/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
