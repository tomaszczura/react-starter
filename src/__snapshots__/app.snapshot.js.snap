// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot initial render 1`] = `
<div>
  Hello world
</div>
`;
